include/deal.II/lac/cuda_vector.h

```diff
@@ -141,10 +141,23 @@ namespace LinearAlgebra
        * copy a vector to the GPU.
        */
       virtual void
+      import_elements(
+        const ReadWriteVector<Number> &V,
+        VectorOperation::values        operation,
+        std::shared_ptr<const Utilities::MPI::CommunicationPatternBase>
+          communication_pattern = {}) override;
+
+      /**
+       * @deprecated Use import_elements() instead.
+       */
+      DEAL_II_DEPRECATED_EARLY virtual void
       import(const ReadWriteVector<Number> &V,
              VectorOperation::values        operation,
              std::shared_ptr<const Utilities::MPI::CommunicationPatternBase>
-               communication_pattern = {}) override;
+               communication_pattern = {}) override
+      {
+        import_elements(V, operation, communication_pattern);
+      }
```

Comment:
I'm being super picky here, but I think it's our general style to not mix definitions and declarations - is there something preventing this definition from being moved to the 'inline functions' part of the file?

Reply:
Oh, that's right, I meant to comment on that and forgot. I was going to say that I am happy to move these functions if that's what people want to do, but that at least for the moment I had implemented them in this way because it makes it easier later to remove these functions once the deprecation period runs out: You only have to touch one place, not two.

But I am happy to do as you suggest if you still feel that way.

Reply:
I vaguely remember some case in which we got extra warnings from defining a deprecated function later on - this works so lets just leave it.